YTPMVE.py

```diff
@@ -123,6 +120,6 @@ def exitScript(errMessage, exitCode, retryWithPythonScript):
     YTPMVE_file.write(str(note_starts[i][0])+",")#Save first argument, note start time
     YTPMVE_file.write(str(note_durations[i])+"\n")#Save second argument, note duration
 
+YTPMVE_file.close()
```

Comment:
looks like this still hasn't been removed, exitScript already handles that

Reply:
Deleted these lines.